core/trino-spi/src/test/java/io/trino/spi/predicate/TestEquatableValueSet.java

```diff
@@ -295,32 +295,36 @@ public void testSubtract()
         assertEquals(EquatableValueSet.of(ID, 0L).complement().subtract(EquatableValueSet.of(ID, 0L, 1L).complement()), EquatableValueSet.of(ID, 1L));
     }
 
-    @Test(expectedExceptions = UnsupportedOperationException.class)
+    @Test
     public void testUnmodifiableCollection()
     {
-        EquatableValueSet.of(ID, 1L).getValues().clear();
+        assertThatThrownBy(() -> EquatableValueSet.of(ID, 1L).getValues().clear())
+            .isInstanceOf(UnsupportedOperationException.class)
+            .hasMessage("cannot clear values");
     }
 
-    @Test(expectedExceptions = UnsupportedOperationException.class)
+    @Test
     public void testUnmodifiableValueEntries()
     {
-        EquatableValueSet.of(ID, 1L).getEntries().clear();
+        assertThatThrownBy(() -> EquatableValueSet.of(ID, 1L).getEntries().clear())
+            .isInstanceOf(UnsupportedOperationException.class)
```

Comment:
Please update the indentation of these lines. 
Please see https://github.com/trinodb/trino/blob/master/DEVELOPMENT.md#code-style for instruction how to import Code style definitions into IntelliJ to have the right amount of indentation.

Reply:
Thanks. I think I was able to import the Code Style definitions into IntelliJ, so do I select "Inspect Code" to check and make sure the code adheres to the style?

Reply:
Once you imported the Airlift code style, you should get the right formatting and indentation by using `Code > Reformat Code`.